Demo/Configuration.swift

```diff
@@ -28,14 +28,14 @@ internal enum ConfigurationConstants {
     
     static let additionalData = ["allow3DS2": true]
 
-    static let clientKey = "{YOUR_CLIENT_KEY}"
+    static let clientKey = "test_I7ABUXA66FCC5OG3FNVMKPVYDIXSYTLR"
     
     static let apiContext = APIContext(environment: componentsEnvironment, clientKey: clientKey)
 
     // swiftlint:disable:next line_length
-    static let demoServerAPIKey = "{YOUR_DEMO_SERVER_API_KEY}"
+    static let demoServerAPIKey = "AQEphmfuXNWTK0Qc+iSDgm02hueYR55DGcMTDWlE1rNRjS1UduCj/CcjoG4QwV1bDb7kfNy1WIxIIkxgBw==-HVMl6ri9a3R23e5A3zeRll3j3ztqGuuDu290HGCqqsw=-p5)54AtT#@d9_8PU"
```

Comment:
@mlink

I would advise you to generate new TEST key.

Reply:
This is a valid point. I forget this is a public repo.

Reply:
@descorp we have generated new API keys. It will take effect in 24 hours. Is it possible for you guys to delete this PR?

Reply:
Hey @zhentan

Not the PR, but you can `amend` this commit and `force-push` or even `delete the branch` from the origin

Reply:
We have deleted the branch already. As you can see if you try to go to the branch. However the PR seems to retain the changed files from the commits.

Reply:
hm.. ok, my bad.
Apparently it doesn't hide the PR 😅
It also since branch is deleted - you can't `amend` commits..

Could you try create branch with same name to check if this will effect the PR?

Reply:
@descorp I force pushed the latest develop branch to this PR branch, it did not affect the PR. Please advice.

Reply:
Probably because it is a PR from fork...
I am afraid there is no "normal" way to delete PR.

The last resort - request GitHub Support
https://support.github.com/contact?tags=rr-remove-data

Important part: if you have created new keys - the old one expires, and there is no security risk 🙂